### DISCUSSION TITLE
How to activate test mode

### DISCUSSION THREAD

**Discussion by b4nst** [cue-lang/cue#1218] [Q&A] [answered]
As I understood, by default cue eval ignore every `_test.cue` files. When trying to specify a `_test.cue` file i get 

```
_test.cue files excluded in non-test mode
```

Which made me wonder, is there a way to be in `test mode`?

**Comment by verdverm** [comment, marked as answer]
#209 #865 

tl;dr `cue test` is planned but not implemented yet, `_test.cue` has been reserved in anticipation.
